apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -9960,6 +10026,13 @@ span#HW_badge_cont:nth-child(3){
     .pagination>li>span {
         padding: 6px 6px;
     }
+    // header .hamburger_menu h4 {
```

Comment:
Remove commented code if not required

Reply:
It will be reuse in future for mobile apk work